plugins/PostProcessingPlugin/scripts/TimeLapse.py

```diff
@@ -66,6 +66,22 @@ def getSettingDataString(self):
                     "type": "float",
                     "default_value": 9000,
                     "enabled": "park_print_head"
+                },
+                "retract":
+                {
+                    "label": "Retraction Distance",
+                    "description": "Filament retraction distance for camera trigger.",
+                    "unit": "mm",
+                    "type": "int",
+                    "default_value": 1
+                },
+                "zhop":
+                {
+                    "label": "Z-Hop when parking",
```

Comment:
```suggestion
                    "label": "Z-Hop Height When Parking",
```

The "normal" z-hop setting is called "Z Hop Height", so it makes sense to use a similar naming.